### PR TITLE
adding a decay heat calculation to material class

### PR DESCRIPTION
**Pull request by FlanFlanagan**

### CODE REVIEW

**Comment by FlanFlanagan** [comment]
@scopatz

File: src/material.cc [hunk 1]
```diff
@@ -193,6 +193,16 @@ void Material::Decay(int curr_time) {
   Transmute(decayed);
 }
 
+double Material::DecayHeat() {
+    double decay_heat = 0.;    
+    pyne::Material p_map = pyne::Material(comp_->atom(), 15.);
```
Comment (scopatz) [inline src/material.cc]: shouldn't this be called `p_mat`? Also what is with the 15?

Comment (scopatz) [inline src/material.cc]: Also, PyNE uses mass weights not atom weights.

File: src/material.cc [hunk 2]
```diff
@@ -193,6 +193,16 @@ void Material::Decay(int curr_time) {
   Transmute(decayed);
 }
 
+double Material::DecayHeat() {
+    double decay_heat = 0.;    
+    pyne::Material p_map = pyne::Material(comp_->mass(), 1.);
```
Comment (FlanFlanagan) [inline src/material.cc]: Ahh I see now

**Comment by scopatz** [comment]
Let me know when you think it is GTG. LGTM Boss

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**